packages/EasyUtils/src/Math.php

```diff
@@ -50,7 +51,9 @@ public function __construct(
 
     public function abs(string $value, ?int $precision = null, ?int $mode = null): string
     {
-        return $this->comp($value, '0') === -1
+        return $this
+            ->compareThat($value)
```

Comment:
Let's keep the previous implementation to not involve the separate class without a real need.

Reply:
Done